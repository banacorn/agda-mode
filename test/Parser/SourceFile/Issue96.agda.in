data MBool : Set where
   TT : MBool
   FF : MBool
   ?? : MBool
   A? : ?
   ?A : ?

?
?
?