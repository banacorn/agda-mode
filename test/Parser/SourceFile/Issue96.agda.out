Hole [0] (73, 74) => (73, 80) "{!   !}"
Hole [1] (89, 90) => (89, 96) "{!   !}"
Hole [2] (98, 99) => (98, 105) "{!   !}"
Hole [3] (106, 107) => (106, 113) "{!   !}"
Hole [4] (114, 115) => (114, 121) "{!   !}"